pkg/operator/controller/dns/controller_test.go

```diff
@@ -0,0 +1,56 @@
+package dns
+
+import (
+	"fmt"
+	"github.com/openshift/cluster-ingress-operator/pkg/dns"
+
+	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	iov1 "github.com/openshift/cluster-ingress-operator/pkg/api/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"testing"
+	"time"
+)
+
+func TestPublishRecordToZones(t *testing.T) {
+
+	var tests = []struct {
+		name   string
+		record *iov1.DNSRecord
+		zone   []configv1.DNSZone
+		result reconcile.Result
+		expect (operatorv1.ConditionStatus)
+	}{
+		{
+			name: "check status",
```

Comment:
The name should be more specific.  This is testing for a successful update of 1 record.